### PR TITLE
Show payment in progress warning message

### PR DESCRIPTION
**Pull request by amanpdyadav**
Refs None
Screenshot 2022-11-21 at 9 52 17

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "common.PurchaseNotification.notification.alert-2.label": "Permit 2 is not yet valid",
   "common.PurchaseNotification.notification.alert-2.message": "Payment is being processed. 2. The vehicle parking permit will take effect automatically once the payment has been processed.",
   "common.PurchaseNotification.notification.alert.label": "Permit not yet valid",
-  "common.PurchaseNotification.notification.alert.message": "Payment is being processed. The parking permit will take effect automatically once the payment has been processed.",
+  "common.PurchaseNotification.notification.alert.message": "Payment is being processed. The parking permit will take effect automatically once the payment has been processed. You can make changes only after the code has entered into force.",
```
Comment (mhieta) [inline src/i18n/en.json:5]: Use:
`Payment is being processed. The parking permit will take effect automatically once the payment has been processed. You can make changes only after the permit has become valid.`

**Review by mhieta — CHANGES_REQUESTED** [review]
Fix english translation.

File: src/i18n/en.json (hunk 1 above)
Comment (amanpdyadav) [reply src/i18n/en.json]: done

**Review by mhieta — APPROVED** [review]
LGTM 👍

**Pull request merged**